nni/nas/oneshot/pytorch/supermodule/sampling.py

```diff
@@ -76,6 +76,14 @@ def reduction(self, items: list[Any], sampled: list[Any]):
         """Override this to implement customized reduction."""
         return weighted_sum(items)
 
+    def _save_module_to_state_dict(self, destination, prefix, keep_vars):
```

Comment:
`_save_module_to_state_dict` -> `save_module_to_state_dict`

Reply:
I think `_save_module_to_state_dict ` is a internal method, it should not be invoked outeside, so i think the '_' before is also ok.